ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -16,12 +16,9 @@
             </p>
           </div>
           <caution-box class="mt-9" :setMsg="cautionTxt" />
-          <registration-length-trust
+          <registration-length-trust-amendment
             class="mt-15"
-            :isSummary="true"
-            @setRegistrationLengthTrustValid="
-              registrationLengthTrustValid = $event
-            "
+            @setRegistrationLengthTrustValid="registrationLengthTrustValid = $event"
```

Comment:
I don't think it's ever invalid... so you can just remove this if you like

Reply:
Removed.